src/nidmm/system_tests/test_system_nidmm.py

```diff
@@ -132,3 +132,10 @@ def test_get_dev_temp(device_info):
         temperature = session.get_dev_temp('')
         print(temperature)
         assert 20 <= temperature <= 50
+
+def test_writeonly_attribute(device_info):
+    with nidmm.Session(device_info['name']) as session:
+        try:
+            session.channel_count = 5
+        except nidmm.Error as e:
+            assert e.code == -1074135027
```

Comment:
According to pep8 (coding convention) there should not be an blank line at the end of the file.

However, having said that, I have disabled that particular check from the flake8 (pep8 enforcement) checks.
1. There were some generated files I couldn't figure out how to keep mako from adding it
1. I don't happen to agree with it

Number 1. was the real reason but 2. was just a happy coincidence.